### ISSUE TITLE
Incorrect string for ignore (blacklist) feature

### ISSUE THREAD

**Issue by Pjotr123** [linuxmint/mintupdate#588]
When right-clicking an available update in the mintupdate window, you have two blacklisting options:

1. `Ignore the current update for this package`

2. `Ignore all future updates for this package`

The second option is wrongly described. It should be:
`Ignore all current and future updates for this package`

.... because that's what its effect is. 

**Distribution:**
Linux Mint 19.2

**Software version:**
5.5.4.2

**Comment by clefebvre** [comment]
Agreed. I don't want to fix this that close to release, our translations are being finalized. But we should do that during the next cycle.
